pmm-app/src/pmm-settings/components/SignUp/FieldAdapters/FieldAdapters.styles.ts

```diff
@@ -0,0 +1,13 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { stylesFactory } from '@grafana/ui';
+
+export const getStyles = stylesFactory((theme: GrafanaTheme) => ({
+  errorMessage: css`
+    color: ${(theme.colors as any).red};
+    font-size: 10px;
+    height: 10px;
+    line-height: 1;
+    margin-top: 3px;
```

Comment:
Use theme values if you think is right instead of hardcoded values (e.g. theme.spacing.sm) and in other files as well

Reply:
Good point, done